specialized/specialized_form_parser.ipynb

```diff
@@ -154,6 +152,44 @@
     "doc = process_document_sample()"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Draw the bounding boxes¶\n",
+    "We will now download the pdf above a jpg and use the spatial data to mark our values."
```

Comment:
"as a jpg"

or maybe more accurate, something like:
"We will now use the spatial data returned by the processor to mark our values on the invoice pdf file that we first converted into a jpg."

Reply:
Done